metoffice_ec2/subset.py

```diff
@@ -12,14 +12,18 @@
 
 def subset(
     dataset: xr.Dataset,
-    height_meters: Optional[List[int]] = None,
+    height_meters: Optional[List[float]] = None,
```

Comment:
I might be wrong but I thought heights were always `ints`?

If they are sometimes `ints` and sometimes `floats` then it might be better to use `List[Number]` (and `from numbers import Number` at the top)

Reply:
Correct me if I am wrong: My understanding is that height_meters here doesn't come from the NetCDF files, but rather from the SNS/SQS messages. I pulled a couple of those, see `data/sns_messages`.
If you look at the `height_meters` on them, all of them see to be float.

Reply:
ok, cool, I was wrong!  Let's leave this as 'float'!